launcher/internal/systemctl/systemctl.go

```diff
@@ -42,6 +43,19 @@ func (s *Systemctl) Stop(unit string) error {
 	return runSystemdCmd(s.dbus.StopUnitContext, "stop", unit)
 }
 
+// GetStatus is the equivalent of `systemctl is-active $unit`.
+// The status can be "active", "activating", "deactivating", "inactive" or "failed".
+func (s *Systemctl) GetStatus(ctx context.Context, unit string) (string, error) {
```

Comment:
IMO this should be called `IsActive` so that it matches the systemctl API.

Reply:
Done.